client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-setup/index.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+import { Gridicon, Button } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import FormattedHeader from 'calypso/components/formatted-header';
+import FormInput from './form-input';
+import type { Step } from '../../types';
+
+import './styles.scss';
+
+const LinkInBioSetup: Step = function LinkInBioSetup( { navigation } ) {
+	const { goNext } = navigation;
+	const siteNameErrorMsg = 'Your site needs a name so your subscribers can identify you.';
+	const siteBriefDescriptionErrorMsg =
+		'Your site needs a brief description so your subscribers can identify you.';
+	const { __ } = useI18n();
+
+	const onChangeSiteName = () => {
+		return;
+	};
+
+	const onChangeSiteDescription = () => {
+		return;
+	};
+
+	return (
+		<div className="step-container">
+			<form>
+				<FormattedHeader align={ 'center' } headerText={ __( 'Set up Link in Bio' ) } />
+				<div className="link-in-bio-setup__form">
+					{ /* replace this component, align with newsletter flow */ }
+					<div className="link-in-bio-setup-form-icon__container">
+						<div className="link-in-bio-setup-form__icon">
+							<Gridicon key={ 'blue' } icon="share-ios" size={ 18 } />
+						</div>
+						<label htmlFor="">{ __( 'Upload site icon' ) }</label>{ ' ' }
+					</div>
+					{ /*  */ }
+					<FormInput
+						inputValue={ '' } // get value from store?
+						isValid={ true }
+						onChange={ onChangeSiteName }
+						errorMessage={ siteNameErrorMsg }
+					/>
```

Comment:
Is it necessary to create a new FormInput for this step? Would be better to use what we already have, extend it if needed and at last, if nothing else works, create new components.
You can take a look at how the `site-options` step is doing [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/site-options/index.tsx#L121)!

Reply:
This is have a good point. I checked this component when I was implementing it... but I noticed that we have one design difference that is the "green check mark" inside the input. I can still use this component and fake it using a wrapper div around it to handle the green check. 

<img width="394" alt="image" src="https://user-images.githubusercontent.com/2653810/182395634-0db582f2-696d-4b02-9236-6b38c2076135.png">

Think will be a good solution. 

Thank you, implementing it.